lib/LMSManagers/LMSEventManager.php

```diff
@@ -242,7 +253,8 @@ public function GetEvent($id)
 			LEFT JOIN customerview c ON (c.id = customerid)
 			LEFT JOIN vusers ON (vusers.id = userid)
 			LEFT JOIN rttickets as rtt ON (rtt.id = e.ticketid)
-			LEFT JOIN netnodes as nn ON (nn.id = rtt.netnodeid)
+			LEFT JOIN netnodes as nn ON (nn.id = e.netnodeid)
```

Comment:
Nigdy przy `JOIN`-ach nie używamy `as`, a tu już zaczynamy?

Reply:
jest to w kilku miejscach ale usunąłem w https://github.com/chilek/lms/pull/2273/commits/6b4aa36f58eb43babfe3c18eb2f60d47e69db7ee
usunę w pozostałych w innym PR